modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -4110,6 +4110,16 @@ def wait_computations(self):
         """Wait for all computations to complete without materializing data."""
         self._partition_mgr_cls.wait_partitions(self._partitions.flatten())
 
+    def support_materialization_in_worker_process(self) -> bool:
+        """
+        Whether it's possible to call function `to_pandas` during the pickling process, at the moment of recreating the object.
+
+        Returns
+        -------
+        bool
+        """
+        return True
```

Comment:
why would we ever want to recreate modin objects on a worker? wouldn't it make applying any method to them super slow anyway?

Reply:
> why would we ever want to recreate modin objects on a worker?

The main goal is to avoid materializing the dataframe in the main process and transfer this operation to the worker process.

> wouldn't it make applying any method to them super slow anyway?

The only operation needed for this is taking the object by reference (`get` operation)  to perform the conversion to pandas and perform any operations on the pandas object.